fitbenchmarking/fitbenchmarking
# Software links in tables

#### Description of Work

Fixes #196 

Adds links to the read the docs how page minimizer list.

#### Testing Instructions

1. Check that all the links work for all minimizers

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
